llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -20756,7 +20769,7 @@ SDValue RISCVTargetLowering::LowerFormalArguments(
         // the vector itself.
         VT = VA.getLocVT();
       }
- 
```

Comment:
Please remove extraneous changes.

Reply:
https://github.com/CHERIoT-Platform/llvm-project/pull/147/commits/3a93c88b3d3f506e0c5fcf094a447d75b54c62c5 removes it.